src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,99 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Entry<K, V>[] table;
+    private int size;
+    private int threshold;
+    private final float loadFactor;
+
+    static class Entry<K, V> {
+        final K key;
+        V value;
+        Entry<K, V> next;
+
+        public Entry(K key, V value, Entry<K, V> next) {
```

Comment:
Don't use getters and setters in class 'Entry' as we have access to private fields of inner class, so using getters or setters is redundant. Please remove the public modifier from the constructor of 'Entry' class.